addon/components/paper-autocomplete.js

```diff
@@ -28,8 +28,13 @@ export default PowerSelect.extend(ValidationMixin, ChildMixin, {
   onblur: computed.alias('onBlur'),
   onchange: null,
   oninput: null,
-  validationProperty: 'searchText',
-
+  validationProperty: computed('onSearchTextChange', 'onSelectionChange', function() {
+    if (this.get('onSearchTextChange')) {
```

Comment:
With this test, we're using `searchText` if both `onSearchTextChange` and `onSelectionChange` are present, which is not what we want to.